packages/xo-server/src/api/vm.js

```diff
@@ -320,6 +320,8 @@ create.params = {
       },
     },
   },
+
+  hvmBootFirmware: { type: ['string', 'null'], optional: true },
```

Comment:
`null` is not needed, better define one value for missing properties.

Reply:
I agree it's not needed for the creation, `null` is used to indicate an explicit removal of a value (ie when editing an object).

Sorry if I told you the contrary in this PR.

Reply:
> `null` is not needed, better define one value for missing properties.

would `unefined` is good to define missing properties ?

Reply:
`undefined` means that the property not exists. But i don't understand the idea behind the question. Can you please provide a suggestion?

Reply:
Sorry, I understand now :)
So if we send an `undefined` property to the server. The server will consider that it doesn't exist.